Added playerAction event (#9)
Lautenschlager-id/Transfromage: docs/Events.md

>Triggered when the player receives a whisper.
>
---
>### playerAction ( playerData, action, flags )
>| Parameter | Type | Description |
>| :-: | :-: | - |
>| playerData | `table` | The data of the player. |

> >| action | `int` | The action id played by the player. |

Reviewer: ```suggestion
>| emote | `enum.emote` | The id of the emote played the player. |
```

> >| flags | `string` | Optional flags. |

Reviewer: ```suggestion
>| flag | `string` | The country code used if the emote is flag. |
```
